easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
```

Comment:
Until https://github.com/hpcugent/easybuild-easyblocks/pull/542 is merged, we should keep using it.